cardano-node/ChangeLog.md

```diff
@@ -25,13 +25,30 @@ None
   format will be supported for next two major releases of the node (the last
   major version which will support it is `1.37`). (#4563)
 
+- Fixed a tight loop when demoting a peer from hot to warm timeouts
```

Comment:
Why are the changes now part of the 1.35.6 release? They were previously part of the 1.35.5 release, not so?

Reply:
These changes were merged after `1.35.5` was released (https://github.com/input-output-hk/cardano-node/commits/release/1.35, `1.35.5` was tagged at 8762a10ef).